src/main/js/posts/comments/comment.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+
+
+class Comment extends React.Component {
+    constructor(props) {
+        super(props);
+    }
+
+    render() {
+        return (
+            <article className='comment-main'>
+                <div className='comment-content'>
+                    {/*{props.comment.content}*/}
```

Comment:
Good point about the `comment.comment_content`, it might be worth renaming this.

The `Comment` class is currently not linked to the database table, it's just a placeholder for when the database and java classes are added.